src/test/java/systems/composable/dropwizard/cassandra/connect/Ec2MultiRegionAddressTranslatorFactoryTest.java

```diff
@@ -0,0 +1,15 @@
+package systems.composable.dropwizard.cassandra.connect;
+
+import org.junit.Test;
+
+import static org.hibernate.validator.internal.util.Contracts.assertNotNull;
+
+public class Ec2MultiRegionAddressTranslatorFactoryTest {
```

Comment:
Please capitalise `EC2` (i.e. no lowercase `c`) to conform to existing naming conventions.